docs/distributions/manjaro/faq.md

```diff
@@ -1,30 +1,60 @@
+# Migrating to new kernel
+
+Run this in your terminal:
+
+```sh
+cat << EOF | sudo tee -a /etc/pacman.conf
+[manjaro-mact2]
+SigLevel = Never
+Server = https://mirror.funami.tech/$repo/os/$arch
+EOF
+sudo pacman -Syu linux516-t2 linux516-t2-headers dkms apple-bce-dkms-git apple-ibridge-dkms-git apple-bcm-wifi-firmware
+sudo pacman -R linux57-mbp 
+sudo pacman -R linux56-mbp
+sudo pacman -R linux57-mbp-headers
+sudo pacman -R linux56-mbp-headers
+sudo mkdir -p /etc/modules-load.d
+sudo touch /etc/modules-load.d/t2.conf
+cat << EOF | sudo tee /etc/modules-load.d/t2.conf
+apple-bce
+brcmfmac
```

Comment:
brcmfmac has modaliases set up so it it's not loading automatically on its own, it should be